packages/dapp/public/index.html

```diff
@@ -7,8 +7,10 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
   <link rel="apple-touch-icon" sizes="180x180" href="%PUBLIC_URL%/apple-touch-icon.png">
-  <link rel="icon" type="image/png" sizes="32x32" href="%PUBLIC_URL%/favicon-32x32.png">
-  <link rel="icon" type="image/png" sizes="16x16" href="%PUBLIC_URL%/favicon-16x16.png">
+  <!-- <link rel="icon" type="image/png" sizes="32x32" href="%PUBLIC_URL%/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="%PUBLIC_URL%/favicon-16x16.png"> -->
+  <link rel="icon" type="image/png" sizes="32x32" href="%PUBLIC_URL%/favicon.ico">
+  <link rel="icon" type="image/png" sizes="16x16" href="%PUBLIC_URL%/favicon.ico">
```

Comment:
We should remove the commented out code if its no longer going to be used. For Plor's sake lol